files/es/web/api/batterymanager/onchargingchange/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: BatteryManager.onchargingchange
+slug: Web/API/BatteryManager/onchargingchange
```

Comment:
Olvidaste agregar las etiquetas,,,
```suggestion
slug: Web/API/BatteryManager/onchargingchange
tags:
  - API
  - Battery API
  - Event Handler
  - NeedsMarkupWork
  - Property
  - Reference
```

Reply:
:fearful: Muchas gracias, @JuanVqz, casi se escapa ese detalle.

Reply:
I know I haven't mentioned anything about tags up to this point, but I think (from one of **SphinxKnight**'s comments that I saw somewhere) tags actually don't have much use in locales. There's of course no issue with adding them, though I wouldn't say excluding them is concerning either :)

Reply:
I was thinking about it, I mean how tags work because if sometimes we translate it and sometimes we don't, it will be a mess, Okay! good to know! You rock, bro :100: